indigo_app/views/places.py

```diff
@@ -765,6 +766,31 @@ def get_success_url(self):
         return reverse('place_settings', kwargs={'place': self.kwargs['place']})
 
 
+class PlaceUsersView(PlaceViewBase, FormView):
+    template_name = 'place/users.html'
+    tab = 'place_users'
+    form_class = PlaceUsersForm
+
+    def get_initial(self):
+        initial = super().get_initial()
+        initial["users"] = [u.id for u in User.objects.filter(badges_earned__slug=f"country-{self.country.code.lower()}")]
+        return initial
+
+    def form_valid(self, form):
+        users = User.objects.all().exclude(is_superuser=True)
```

Comment:
```suggestion
        users = User.objects.all()
```

Reply:
superusers still need country badges in order to be on the list of users that can be assigned a task in a country